Add VAST plugin (#23)
tenzir/threatbus: apps/vast/README.md

Threat Bus to VAST Bridge
=========================

> Threat Bus is a publish-subscribe broker for threat intelligence. It is expected
> that applications register themselves at the bus. Since vast cannot do so on its
> own (yet), the `vast-bridge.py` is used for that purpose.

Reviewer: ```suggestion
Threat Bus is a publish-subscribe broker for threat intelligence. It is expected
that applications register themselves at the bus. Since VAST cannot do so on its
own (yet), `vast-bridge.py` implements that functionality in the meantime.
```

> The bridge provides a thin layer around the PyVAST CLI bindings. It facilitates

Reviewer: ```suggestion
The bridge provides a thin layer around 
[PyVAST](https://docs.tenzir.com/vast/python-bindings/overview), VAST's Python CLI bindings. It facilitates
```

> message exchange between Threat Bus and a vast instance.

Reviewer: ```suggestion
message exchange between Threat Bus and a VAST instance.
```

## Setup

> - Install the PyVAST CLI bindings. See [github.com/tenzir/vast/README.md](https://github.com/tenzir/vast/blob/master/pyvast/)

Reviewer: ```suggestion
- [Install PyVAST](https://docs.tenzir.com/vast/python-bindings/installation)
```

> - Install python bindings for ZeroMQ

Reviewer: ```suggestion
- Install Python bindings for ZeroMQ
```

```sh
virtualenv --system-site-packages venv
source venv/bin/activate
python3 -m pip install -r requirements.txt
```

## Usage

The bridge gets configured via command line arguments and flags. Start the
bridge as follows.

```sh

> ./vast-bridge --vast-binary=/opt/tenzir/bin/vast --vast=localhost:42000 --threatbus=localhost:13370

Reviewer: Can `--vast-binary` be made optional so that it takes `vast` from `$PATH`?

Author: That is already the case. The default option for the `--vast-binary` is simply `vast`, which will then be searched in `$PATH`
